lib/queue.js

```diff
@@ -2,11 +2,16 @@
 
 var async = require('async')
 var chalk = require('chalk')
```

Comment:
except nobody will be using the debug flag in production, nor frequently. (think of the use case where the agent is actually running on a client's server, highly unlikely a user will go in and enable debug flag and watch messages in production)

this is for your own usage right now, but for clients using the agent, its adding little value.

lets keep the dependencies at a minimum.